### DISCUSSION TITLE
Can I start Volume Mixer from command line?

### DISCUSSION THREAD

**Discussion by mockroot** [File-New-Project/EarTrumpet#641] [Ideas]
Hi. Can I start this window from command line?
![screenshot_05-08-2018_06-14-33](https://user-images.githubusercontent.com/24863704/43682228-0242c6fa-9877-11e8-9e6a-998ba505456f.png)

I want to create shortcut for this window with AutoHotkey. If there is a command line option, it would be great. Thank you.

**Comment by MarcG2** [comment]
I would have also found this useful.  If your using AHK a simple work around would be to have AHK send the keyboard hotkey that eartrumpet is configured to use.  I had my AHK script using the default hotkey which was removed at some point.  

Anyway where are Eartrumpet settings saved at?  I would like to set my eartrumpet keyboard shortcuts using a script.  I own many computers and having to manually set the shortcut every time I do a fresh windows setup is annoying.

**Comment by NIronwolf** [comment]
I would also love this feature.  It would be perfect if I could specify a specific device to have open too.

**Reply by ckolumbus** [reply]
me2 🙋‍♂️

**Comment by shodanx2** [comment]
Hello,
The issue was closed and locked as completed
I wish to create a shortcut to directly open the volume mixer
Or at least make the volume mixer the default instead of the flyout mixer which I do not like

If it is completed, what is the command line argument to start open the volumixer ?
